rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1395,6 +1392,120 @@ void should_ThrowException_When_CancelClaimingOfClaimedTaskByAnotherUser() {
+    assertThat(requestReviewResponse.getStatusCode()).isEqualTo(HttpStatus.OK);
+    repModel = requestReviewResponse.getBody();
+    assertThat(repModel.getOwner()).isNull();
+    assertThat(repModel.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+  }
+
+  @Test
+  void should_ForceRequestReviewOnATask() {
+    String url =
+        restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:000000000000000000000000000000000101");
+    HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("user-1-1"));
+
+    // retrieve task from Rest Api
+    ResponseEntity<TaskRepresentationModel> getTaskResponse =
+        TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+    assertThat(getTaskResponse.getBody()).isNotNull();
+    TaskRepresentationModel repModel = getTaskResponse.getBody();
+    assertThat(repModel.getState()).isEqualTo(TaskState.CLAIMED);
+    assertThat(repModel.getOwner()).isEqualTo("user-1-2");
+
+    // request review
+    String url2 =
+        restHelper.toUrl(
+            RestEndpoints.URL_TASKS_ID_REQUEST_REVIEW_FORCE,
+            "TKI:000000000000000000000000000000000101");
+    ResponseEntity<TaskRepresentationModel> requestReviewResponse =
+        TEMPLATE.exchange(url2, HttpMethod.POST, auth, TASK_MODEL_TYPE);
+
+    assertThat(requestReviewResponse.getBody()).isNotNull();
+    assertThat(requestReviewResponse.getStatusCode()).isEqualTo(HttpStatus.OK);
+    repModel = requestReviewResponse.getBody();
+    assertThat(repModel.getOwner()).isNull();
+    assertThat(repModel.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+  }
+
+  @Test
+  void should_RequestChangesTask() {
+    String url =
+        restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:000000000000000000000000000000000136");
+    HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("user-1-1"));
+
+    // retrieve task from Rest Api
+    ResponseEntity<TaskRepresentationModel> getTaskResponse =
+        TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+    assertThat(getTaskResponse.getBody()).isNotNull();
+    TaskRepresentationModel repModel = getTaskResponse.getBody();
+    assertThat(repModel.getState()).isEqualTo(TaskState.IN_REVIEW);
+    assertThat(repModel.getOwner()).isEqualTo("user-1-1");
+
+    // request changes
+    String url2 =
+        restHelper.toUrl(
+            RestEndpoints.URL_TASKS_ID_REQUEST_CHANGES, "TKI:000000000000000000000000000000000136");
+    ResponseEntity<TaskRepresentationModel> requestedChangesResponse =
+        TEMPLATE.exchange(url2, HttpMethod.POST, auth, TASK_MODEL_TYPE);
+
+    assertThat(requestedChangesResponse.getBody()).isNotNull();
+    assertThat(requestedChangesResponse.getStatusCode()).isEqualTo(HttpStatus.OK);
+    repModel = requestedChangesResponse.getBody();
+    assertThat(repModel.getOwner()).isNull();
+    assertThat(repModel.getState()).isEqualTo(TaskState.READY);
+  }
+
+  @Test
+  void should_ForceRequestChangesTask() {
```

Comment:
What do you think about should_ForceRequestChanges_When_CurrentUserIsNotTheOwner?

Reply:
done